quarterback-packages/track-changes-plugin/src/mutate/deleteNode.ts

```diff
@@ -42,7 +42,7 @@ export function deleteNode(node: PMNode, pos: number, tr: Transaction) {
   // Block nodes can be deleted by just removing their start token which should then merge the text
   // content to above node's content (if there is one)
   const canMergeToNodeAbove =
-    (resPos.parent !== tr.doc || resPos.nodeBefore) && node.isBlock && node.firstChild?.isText
+    resPos.parent !== tr.doc && resPos.nodeBefore && node.isBlock && node.firstChild?.isText
```

Comment:
The `replaceWith` call in Line:47 will not work in case you are going to delete the first child, of the keyword group.

To clarify the issue here related to deleting the first node in a group so this issue surfaced now as we can't delete the first element in a section which is the title. and the root cause of this issue starts from the [mergeNode](https://github.com/Atypon-OpenSource/manuscripts-quarterback/blob/master/quarterback-packages/track-changes-plugin/src/mutate/mergeNode.ts#L32) the join will remove the second child in case you want to remove the first child

Reply:
I see, please add that as a comment. The quaterback is hard to understand so any explanation of what we do there in the code will help. Thanks

Reply:
updated